test/models/config_test.rb

```diff
@@ -94,5 +94,25 @@ class ConfigTest < ActiveSupport::TestCase
         assert(Config.start_day.equal? :tuesday)
       end
     end
+
+    describe '#hide_practical_support?' do
+      it 'can return true' do
+        c = Config.find_or_create_by(config_key: 'hide_practical_support')
+        c.config_value = { options: ["Yes"] }
+        c.save!
+        assert(Config.hide_practical_support? == true)
```

Comment:
apologies for the ruby style nitpick, but `assert Config.hide_practical_support?` and (`assert_not Config.hide_practical_support?`) is a bit tidier than `assert boolean == true`. If it's not a ton of trouble I'd appreciate making that bump for consistency but am also happy to do it since this is my own weird neurosis, haha.

Reply:
So the reason I did it this way was to enforce that it is returning a proper true/false instead of truthy/falsey, happy to change this if we don't want to enforce that strictly though!

Reply:
If we were very specifically looking for a `true` boolean value here I think we'd want to use `assert_equal`, but I don't think we need to strictly enforce this and checking for truthiness/falsiness is fine. That said I want this PR in, so going to merge.